indigo_app/templates/place/detail.html

```diff
@@ -5,6 +5,38 @@
 
 {% block content %}
 <div class="container mt-3 mb-5">
+  <div class="mr-auto">
+    {% if latest_completeness_stat %}
+      <h6 class="">Core Details</h6>
+      <div class="d-flex">
+        <div class="p-relative mr-2" style="height: 50px; width: 150px;">
+          <canvas id="completeness-chart" data-values="{{ completeness_history|join:',' }}"></canvas>
```

Comment:
The JS from the place work view that renders this chart must be moved over to this view's JS.